recordTimeWorked.py

```diff
@@ -33,23 +37,25 @@ def fill_form(description, date, duration, tags):
     submit = browser.find_element_by_id('sysverb_insert_bottom')
     #input data
     in_task.clear()
-    if (tags == 'pd'):
-        in_category.select_by_index(2)
-    elif (tags == 'ooo'):
+    if (tags == 'ooo'):
         in_category.select_by_index(1)
+    elif (tags == 'pd'):
+        in_category.select_by_index(2)
     elif (tags == 'conv'):
         in_category.select_by_index(3)
+    elif (tags == 'cler'):
+        in_category.select_by_index(4)
     else:
         in_task.send_keys(tags)
     time.sleep(1)
     in_date.clear()
     in_date.send_keys(date)
     time.sleep(1)
     in_time_hours.clear()
-    in_time_hours.send_keys(duration[0])
+    in_time_hours.send_keys(duration[0:2])
```

Comment:
I noticed the field in the csv for the duration had a hidden leading 0 which threw off the index of characters for keying the hours and minutes fields